source/objects/responseClass.m

```diff
@@ -318,6 +318,141 @@ function plotForces(obj,bodyNum,comp)
             clear t FT FE FRD FR FV FM i
         end
         
+        function plotWaves(obj,it,tsf,g,axis_lims,body,waves,save)
+            % This method plots the wave elevation and body geometry over
+            % time to visualize the waves and response
+            %
+            % Parameters
+            % ------------
+            %     it : float
+            %         simulation time step (simu.dt)   
+            %
+            %     tsf : float
+            %         (time step factor) number of simulation time steps
+            %         per video frame (higher number decreases computation
+            %         time)
+            %
+            %     g : float
+            %         acceleration due to gravity (simu.g)
+            %
+            %     axis_lims : 1x6 float matrix
+            %         upper and lower limits to the figure
+            %         [xmin xmax ymin ymax zmin zmax]
```

Comment:
If we require users to input the simu object, then we can avoid taking it, g and axis_lims as parameters. Instead simu.g, simu.domain and simu.dt can be used directly. Also I would recommend renaming some variables as I'll note below. If possible keep variable names the same as the parameter that will likely be input to keep things clear. Also use camel case per WEC-Sim's convention.